Update getting-started.adoc (#131)
fedora-silverblue/silverblue-docs: modules/ROOT/pages/getting-started.adoc

These include:

* Flatpak apps: this is the primary way that (GUI) apps get installed on {variant-name}.

> * Toolbox: Used primarily for CLI apps; development, debugging tools, etc, but also has support for graphical apps

Reviewer: Nit: Missing a `.` here.

> * Package layering: Most Fedora packages can be installed to the system with the help of package layering. By default the system operates in pure image mode, but package layering is useful for things like libvirt, drivers, etc.

Reviewer: Let's keep the line break (+small nit):

```suggestion
* Package layering: Most Fedora packages can be installed on the system with the help of package layering.
  By default the system operates in pure image mode, but package layering is useful for things like libvirt, drivers, etc.
```

Once the Flatpak is downloaded, the Software application will open a new window with an “Install” button (2).
Click this button to begin installation.

Alternatively, each application on flathub.org can be installed through the terminal by running the installation command at the bottom of the page that should look something like this:

 $ flatpak install flathub <package-name>
 

> As an example, Spotify can be installed by running the following command which can be found on Spotify's flathub page:

Reviewer: Let's pick a popular open source software instead.

 $ flatpak install flathub com.spotify.Client

=== Flatpak command line

> Additional details when it comes to using the flatpak command line interface can be found on the official http://docs.flatpak.org/en/latest/using-flatpak.html[Flatpak documentation].

Reviewer: ```suggestion
Additional details about the flatpak command line interface can be found in the official http://docs.flatpak.org/en/latest/using-flatpak.html[Flatpak documentation].
```

Most (but not all) RPM packages provided by Fedora can be installed on {variant-name} using this method.

Currently, using package layering creates a new "deployment", or bootable filesystem root.

> It does not affect your current root. This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated, unless you run `rpm-ostree install --apply-live <pkg>`, in which case it applies it directly to your current deployment. 

Reviewer: ```suggestion
It does not affect your current root. This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated.
You can alternatively use `rpm-ostree install --apply-live <pkg>` to also temporarily apply the change directly to your currently booted deployment.
```

Currently, using package layering creates a new "deployment", or bootable filesystem root.
It does not affect your current root.

> This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated.

Reviewer: Can you add a mention to `rpm-ostree install --apply-live <pkg>`? Thanks!

Author: Wasn't that an experimental feature?

Reviewer: This one is stable now: https://github.com/coreos/rpm-ostree/releases/tag/v2022.13
